libraries/AP_DAL/AP_DAL.h

```diff
@@ -173,6 +173,17 @@ class AP_DAL {
         return _RFRN.fly_forward;
     }
 
+    bool get_takeoff_expected(void) const {
+        return _RFRN.takeoff_expected;
+    }
+
+    bool get_touchdown_expected(void) const {
+        return _RFRN.touchdown_expected;
+    }
+
+    // for EKF usage to enable takeoff expected
+    void set_takeoff_expected();
```

Comment:
I wonder if this is confusing to have two versions of this parameter.. maybe rename to "set_takeoff_expected_to_true()"?

Reply:
i did the comment change instead